core/src/main/java/org/mskcc/cbio/portal/util/CancerStudyPermissionEvaluator.java

```diff
@@ -205,11 +205,13 @@ private boolean hasPermission(CancerStudy cancerStudy, Authentication authentica
         if (GlobalProperties.filterGroupsByAppName()) {
             for (String au : allAuthorities) {
                 if (au.toUpperCase().startsWith(appName + ":")) {
-                    grantedAuthorities.add(au.substring(appName.length() + 1));
+                    grantedAuthorities.add(au.substring(appName.length() + 1).toUpperCase());
```

Comment:
I haven't had time to look at this in depth, does changing to uppercase have the potential to break existing authorites tables?